code/datums/craft/recipes/floor.dm

```diff
@@ -1,254 +1,119 @@
+	result = /obj/item/stack/tile/floor_freezer
 
-/datum/craft_recipe/floor/dark/panels
-	name = "dark panel tile"
-	result = /obj/item/stack/tile/floor/dark/panels
+// FLOOR WHITE
 
-/datum/craft_recipe/floor/dark/techfloor
-	name = "dark techfloor tile"
-	result = /obj/item/stack/tile/floor/dark/techfloor
+/datum/craft_recipe/floor/white
+	name = "white tile"
+	result = /obj/item/stack/tile/floor_white
 
-/datum/craft_recipe/floor/dark/techfloor_grid
-	name = "dark techfloor tile with vents"
-	result = /obj/item/stack/tile/floor/dark/techfloor_grid
+// GRID
 
-/datum/craft_recipe/floor/dark/brown_perforated
-	name = "dark brown perforated tile"
-	result = /obj/item/stack/tile/floor/dark/brown_perforated
+/datum/craft_recipe/floor/grid
+	name = "grey grid tile"
+	result = /obj/item/stack/tile/grid
 
-/datum/craft_recipe/floor/dark/gray_perforated
-	name = "dark gray perforated tile"
-	result = /obj/item/stack/tile/floor/dark/gray_perforated
+// LINOLEUM
 
-/datum/craft_recipe/floor/dark/cargo
-	name = "dark cargo tile"
-	result = /obj/item/stack/tile/floor/dark/cargo
+/datum/craft_recipe/floor/linoleum
+	name = "linoleum floor tile"
+	result = /obj/item/stack/tile/linoleum
+	steps = list(
+		list(CRAFT_MATERIAL, MATERIAL_PLASTIC, 1),
+	)
+	time = 1 //Crafting individual tiles is fast
 
-/datum/craft_recipe/floor/dark/brown_platform
-	name = "dark brown platform tile"
-	result = /obj/item/stack/tile/floor/dark/brown_platform
+// RIDGE
 
-/datum/craft_recipe/floor/dark/gray_platform
-	name = "dark gray platform tile"
-	result = /obj/item/stack/tile/floor/dark/gray_platform
+/datum/craft_recipe/floor/ridge
+	name = "ridge floor tile"
+	result = /obj/item/stack/tile/ridge
 
-/datum/craft_recipe/floor/dark/danger
-	name = "dark danger tile"
-	result = /obj/item/stack/tile/floor/dark/danger
+// TECH
 
-/datum/craft_recipe/floor/dark/golden
-	name = "dark golden tile"
-	result = /obj/item/stack/tile/floor/dark/golden
+/datum/craft_recipe/floor/tech
+	name = "tech grey floor tile"
+	result = /obj/item/stack/tile/techgrey
 
-/datum/craft_recipe/floor/dark/bluecorner
-	name = "dark blue corner tile"
-	result = /obj/item/stack/tile/floor/dark/bluecorner
+/datum/craft_recipe/floor/tech/grid
+	name = "tech grid floor tile"
+	result = /obj/item/stack/tile/techgrid
 
-/datum/craft_recipe/floor/dark/orangecorner
-	name = "dark orange corner tile"
-	result = /obj/item/stack/tile/floor/dark/orangecorner
+/datum/craft_recipe/floor/tech/maint
+	name = "panels maint floor tile"
+	result = /obj/item/stack/tile/techmaint
 
-/datum/craft_recipe/floor/dark/cyancorner
-	name = "dark cyan corner tile"
-	result = /obj/item/stack/tile/floor/dark/cyancorner
+// WOOD
 
-/datum/craft_recipe/floor/dark/violetcorener
-	name = "dark violet corener tile"
-	result = /obj/item/stack/tile/floor/dark/violetcorener
+/datum/craft_recipe/floor/wood
+	name = "wood floor tile"
+	result = /obj/item/stack/tile/wood
+	steps = list(
+		list(CRAFT_MATERIAL, 1, MATERIAL_WOOD)
```

Comment:
These need to go:
CRAFT_MATERIAL, materialname, quantity, worktime

you got it right previously, its ordered wrong here

Reply:
Oops.